tutorial/source/bayesian_regression.ipynb

```diff
@@ -92,333 +109,367 @@
-      "text/plain": [
-       "<Figure size 864x432 with 2 Axes>"
-      ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
      "output_type": "display_data"
     }
    ],
    "source": [
     "fig, ax = plt.subplots(nrows=1, ncols=2, figsize=(12, 6), sharey=True)\n",
-    "african_nations = data[data[\"cont_africa\"] == 1]\n",
-    "non_african_nations = data[data[\"cont_africa\"] == 0]\n",
+    "african_nations = df[df[\"cont_africa\"] == 1]\n",
+    "non_african_nations = df[df[\"cont_africa\"] == 0]\n",
     "sns.scatterplot(non_african_nations[\"rugged\"], \n",
-    "            np.log(non_african_nations[\"rgdppc_2000\"]), \n",
+    "            non_african_nations[\"rgdppc_2000\"], \n",
     "            ax=ax[0])\n",
     "ax[0].set(xlabel=\"Terrain Ruggedness Index\",\n",
     "          ylabel=\"log GDP (2000)\",\n",
     "          title=\"Non African Nations\")\n",
     "sns.scatterplot(african_nations[\"rugged\"], \n",
-    "            np.log(african_nations[\"rgdppc_2000\"]), \n",
-    "            ax=ax[1])\n",
+    "                african_nations[\"rgdppc_2000\"], \n",
+    "                ax=ax[1])\n",
     "ax[1].set(xlabel=\"Terrain Ruggedness Index\",\n",
     "          ylabel=\"log GDP (2000)\",\n",
-    "          title=\"Afri
```

Comment:
We don't need to register for unconstrained parameters. The original `nn.Parameter` also works.

Reply:
This comment is for the definition of `LinearRegression` class. I don't know why it displays a different position here.

Reply:
You mean we can use `nn.Parameter` instead of `PyroParam`? That's true, but I wanted to write something that would work with constraints too. I'm happy to remove it if it seems confusing.

Reply:
Done.